src/plugins/util/OverlayPlugin.js

```diff
@@ -26,6 +26,13 @@ module.exports = class OverlayPlugin {
                 this.closeOverlay();
             }
         });
+
+        // Attach listener for first object created only
+        if (!OverlayPlugin.hasBeenCalled) {
```

Comment:
it's a kind of static method for a Class, this way we attach the listener only once